### ISSUE TITLE
Transfer ownership

### ISSUE THREAD

**Issue by trq** [antimattr/mongodb-migrations#16]
As this project appears to no longer be maintained I am requesting that I take ownership and that this repository is relocated over to my account.

I am actively using this package, but if you know anyone else that might be interested, go for it.

I would then mark this package as deprecated on packagist and suggest trq/mongodb-migrations in it's place.

**Comment by trq** [comment]
Should probably poke @rcatlin in this. Are you interested in taking this up?

**Comment by caciobanu** [comment]
I am also actively using this package and I am interested in maintaining it.

My suggestion is to move this repository to the [doctrine](https://github.com/doctrine) organization.

**Comment by trq** [comment]
I'm all for that too. Not really interested in "taking over" per say, but it would be nice to get some of these PRs merged and get this thing actively maintained.

Should we just ping some people from Doctrine ?

It should be mentioned that there is a bundle over here too: https://github.com/antimattr/mongodb-migrations-bundle/issues/4

**Comment by trq** [comment]
@malarzm Do you know if/how we could get this to happen?

**Comment by malarzm** [comment]
Generally speaking @mikeSimonson is taking care of doctrine/migrations so it'd be good to consult with him, as for taking ownership of project best would be pinging @beberlei or @Ocramius however I don't think it'll happen as there's already quite a lot of work with projects we own :)

**Comment by Ocramius** [comment]
From my PoV, we're trying to reduce maintenance work, not increase it :-|

**Comment by trq** [comment]
Fair enough. I just want to get this somewhere where it might be
maintained. I don't mind doing that maintenance myself or getting others
involved.

**Comment by caciobanu** [comment]
You can count me in for maintenance.

**Comment by caciobanu** [comment]
@matthewfitz , @rcatlin we would love to get some feedback from you guys.

**Comment by trq** [comment]
Seems pretty clear this project is dead, there are numerous pr's just sitting there. Where do we go from here?

**Comment by caciobanu** [comment]
I'm not sure if it makes sense to create an organization just for 2 repos. Or maybe in the future we will have more mongodb related projects? What do you say?

**Comment by trq** [comment]
If there are a few wanting to contribute an org makes perfect sense. It's
not like it's more work/cost to maintain. I'd be very happy to be part of a
small org.

**Comment by caciobanu** [comment]
Then we can go with an org. I'm offering to help or make the org but for the moment I am on vacation untill 22nd of August with limited internet access.

**Comment by trq** [comment]
The biggest issue at the moment is we really need contact from the current
maintainers.

**Comment by caciobanu** [comment]
@trq maybe we should just create new repositories from these instead of waiting for an answer? Seems hopeless to wait.

**Comment by redthor** [comment]
I created doesntmattr :)

**Comment by redthor** [comment]
FYI

I've applied the open PRs to:

https://github.com/doesntmattr/mongodb-migrations-bundle
https://github.com/doesntmattr/mongodb-migrations

**Comment by trq** [comment]
I'm not sure just moving the package somewhere else is all that effective.
The current package needs to be deprecated on packagist.

Of course, it's better than waiting around for a response which doesn't
seem to be coming.

**Comment by redthor** [comment]
yes, we can just switch our composer files and keep going. You're right though, it would be good if it were deprecated on packagist.

**Comment by caciobanu** [comment]
Maybe instead of cloning we should import history to a new repo ?

```
git remote add new repo-url
git push --all new
git push --tags new
```

Also maybe we can reach someone to help us deprecate the old composer package and make it point to the new repo ?

@Seldaek is there a way to do the former ?

**Comment by redthor** [comment]
@caciobanu 
just moved the clone v. new repo to the referenced ticket. More activity in the other repos will help with redirecting people there. We should open new tickets on those repos.

**Comment by Seldaek** [comment]
I could update the package on packagist to mark it abandoned or even update its url to point to your new repo, but I'd rather not do that without agreement from the owners.. I'll send an email to @matthewfitz and see.

**Comment by caciobanu** [comment]
Thank you for your input @Seldaek. We hope that @matthewfitz will answer.

**Comment by redthor** [comment]
@Seldaek - any luck reaching @matthewfitz ?

**Comment by Seldaek** [comment]
Nope no reply :/ If I have to intervene at all I'd rather limit it to marking this package abandoned and redirect people to whatever alternative seems best suited. That way it's easily reversible should Matthew wish to get control back.

**Comment by jwage** [comment]
I will talk to @matthewfitz. Can we just add you to the project repo as a collaborator with full access to manage the project?

**Comment by jwage** [comment]
We still actively use this library at @OpenSky everyday too so we are interested in the project moving forward as well. I reached out to @matthewfitz about adding you all as collaborators.

**Comment by redthor** [comment]
@jwage you could start using the https://github.com/doesntmattr repos - they are on packagist.

We have merged in all the open PRs including symfony 3.x support, switching over php-cs-fixer to friendsofphp and we've also fixed and updated a few other things. There is now version 1.1 for both the library and bundle.

Of course if you manage to get in touch with @matthewfitz then we can just merge that work back. 

Lastly we can other people to the https://github.com/doesntmattr team if they want?

**Comment by jwage** [comment]
I think it will be best if we keep the library here and I imagine Matt will want to keep the lib in his org as well. I work with Matt so I will for sure be able to get this squared away on Monday.

**Comment by caciobanu** [comment]
It would be great to sort this out. Let's wait for Monday and see what happens.

**Comment by caciobanu** [comment]
@jwage any news ?

**Comment by redthor** [comment]
It'd be good to close this off. @jwage - have you had a chance to catch up with @matthewfitz ?
thanks

**Comment by redthor** [comment]
Merry Christmas! :christmas_tree: 
Is there an @OpenSky party or final get together that this could be mentioned?
thanks

**Comment by redthor** [comment]
Just letting the thread know there's a new feature ( execute --replay ) over here: https://github.com/doesntmattr/mongodb-migrations/pull/12

**Comment by manuelbcd** [comment]
Hi. Is finally the packagist reference pointed to doesntmatter fork?  
In doesntmatter the installation instructions describes to add to composer.json

        "antimattr/mongodb-migrations": "~1.0@stable",
        "antimattr/mongodb-migrations-bundle": "~1.0@stable"

Is this correct?

**Comment by caciobanu** [comment]
No. use "doesntmattr/mongodb-migrations-bundle": "~1.0@stable". I will update the docs today.

**Comment by tuscanicz** [comment]
OK, reading the discussion here - we have decided to use @redthor fork doesntmattr/mongodb-migrations-bundle. I know it is not the best solution, but we can't wait years for Matthew to start acting. Thank you all for your contributions!

**Comment by redthor** [comment]
I've had github support disconnect the forks so when we merge PRs into "doesntmattr/mongodb-migrations" it won't prompt to merge them back to antimattr.

@Seldaek do you think you we should mark this package as 'abandoned'?

thanks

**Comment by Seldaek** [comment]
If it is abandoned then yes probably. Sorry but not going to read the whole backlog here.. If you have a specific question please be more specific.

**Comment by redthor** [comment]
`v2.0` of https://github.com/doesntmattr/mongodb-migrations and the bundle has been released.
